src/components/rewind/rewind.js

```diff
@@ -65,14 +66,15 @@ class Rewind extends Component {
     return (
       <div className={[style.controlButtonContainer, style.noIdleControl].join(' ')}>
         <Tooltip label={this.props.rewindText}>
-          <button
+          <Button
+            type={'button'}
```

Comment:
same Q in forward.js

Reply:
I missed that, they were from first PR, I forgot to remove them